cms/src/main/java/org/onehippo/taxonomy/plugin/TaxonomyPickerPlugin.properties

```diff
@@ -0,0 +1 @@
+edit = Select terms
```

Comment:
Do you mean in different brcloud environments ? If so, I'll take this out, as we can specify properties if we make use of the Parameterised classes (which can be used even on the default Taxonomy field for a document type)

Reply:
Yes, on brCloud environments

Reply:
OK, in which case, we'll have to switch this to use the Parameterised version. It can be passed a string to add to the button in preference to the one passed in as a property. I'll update this